_layouts/base.html

```diff
@@ -119,7 +119,7 @@ <h1><a href="{{ site.baseurl }}/">{{ site.title }}</a></h1>
             crossorigin="anonymous"></script>
     <script type="text/javascript">
       anchors.options.placement = 'left';
-      anchors.add('.page > h2, .page > h3, .page > h4, .page > h5, .page > h6');
+      anchors.add('#page > h2, #page > h3, #page > h4, #page > h5, #page > h6');
```

Comment:
Out of interest why not just add this to all h2 etc. rather than filtering to `page`?